Makefile

```diff
@@ -166,6 +173,27 @@ wait-kubernetes-dependencies: ## Wait for dependencies to be ready
 	  --selector=app.kubernetes.io/component=controller \
 	  --timeout=90s
 
+update-version: ## update release version
+	find . -type f | xargs sed -i 's/$(VERSION)/${RELEASE_VERSION}/g'
+
+bump-version: ## update post-release version and update commit message
+	find . -type f | xargs sed -i 's/$(RELEASE_VERSION)/${NEXT_VERSION}/g'
+
+git-commit: ## adds all files and bumps the version
+	git add .
+	git commit -m 'Version bump'
```

Comment:
Can we include also the version we bump to? (should ease the search when/if needed).

Reply:
Sure we can. I based the flow on current way of releasing. It makes sense to have version there.

Reply:
Overall lgtm, only one question: is it ok that the commit is unsigned?